nearcore/src/config.rs

```diff
@@ -334,6 +334,43 @@ fn default_enable_rocksdb_statistics() -> bool {
     false
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct StoreConfig {
```

Comment:
Alternatively make `near_store::StoreConfig` serialisable and refer to it directly. You can mark `read_only` field in the struct as `serde(skip)` and thus it won’t be present in the config.json. This reduces amount of code.